Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -742,6 +749,18 @@ void Wiimote::Update()
 			}
 		}
 
+		const ControlState sideways_toggle_pressed = m_hotkeys->controls[0]->control_ref->State();
+		const ControlState upright_toggle_pressed = m_hotkeys->controls[1]->control_ref->State();
+
+		if (sideways_toggle_pressed > s_sideways_toggle_last)
+			m_options->settings[1]->value = !(m_options->settings[1]->value);
+
+		if (upright_toggle_pressed > s_upright_toggle_last)
+			m_options->settings[2]->value = !(m_options->settings[2]->value);
+		
```

Comment:
What is it complaining about, exactly? The output doesn't make any sense to me:

```
r=$(find Source/Core -name "*.cpp" -o -name "*.h" | xargs egrep -n "\s+$");if ! [ -z "$r" ]; then echo "$r"; exit 1; else echo "OK"; fi
 in dir /home/fifoci/buildslave/lint/build (timeout 1200 secs)
 watching logfiles {}
 argv: r=$(find Source/Core -name "*.cpp" -o -name "*.h" | xargs egrep -n "\s+$");if ! [ -z "$r" ]; then echo "$r"; exit 1; else echo "OK"; fi
 using PTY: False
Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp:760:		
program finished with exit code 1
elapsedTime=1.796332
```

Is it just not liking that I have line 760 as a line of whitespace or something? It becomes harder to read without it, but I can remove it, I guess..